view/allocation_tree.xml

```diff
@@ -4,4 +4,5 @@ this repository contains the full copyright notices and license terms. -->
 <tree>
     <field name="employee"/>
     <field name="work"/>
+    <field name="company"/>
```

Comment:
I would not add this field as it's only used in a o2m from within the work, which already has a company.